src/main/resources/api/service_openapi.yaml

```diff
@@ -393,7 +404,7 @@ components:
     MemberEmail:
```

Comment:
Maybe  change to `GrantRoleRequest` or similar, as more fields may be added in future.